### ISSUE TITLE
Unclear license

### ISSUE THREAD

**Issue by jtojnar** [SSilence/selfoss#1218]
The README and other documents state:

> selfoss was created by Tobias Zeising and it is licensed under the GPLv3 license.

But it is not clear if this was meant to be only version 3 of the license, or also allow later versions of GPL, as [strongly recommended by GNU foundation](https://www.gnu.org/licenses/gpl-faq.html.en#VersionThreeOrLater) – people often use this phrasing, considering the upgradability obvious.

See also https://gnu.org/licenses/identify-licenses-clearly.html

In order to clear up this uncertainty, we will need the past contributors to confirm if they are fine with licensing their contribution under GPL 3 or later, or remove their contributions.

cc @SSilence @niol

**Comment by SSilence** [comment]
👍 version 3 and newer are perfect!

**Comment by jtojnar** [comment]
Also note that this only concerns the license of the source code of selfoss itself. The license of the resulting application bundle will depend on the licenses of our dependencies and we will need to do a audit to figure it out :scream_cat: 

And just to be clear and explicit:

I agree with licensing my contributions to selfoss under GNU GPL 3 or any later version.

**Comment by jtojnar** [comment]
At least the Weblate translations are marked GPL3+ so that reduces the fraction of [contributors](https://github.com/fossar/selfoss/graphs/contributors) we need to ask.

**Comment by niol** [comment]
GPL3+ is good for me.

**Comment by dubbl** [comment]
Hi, I made some commits back in the day ( https://github.com/fossar/selfoss/pull/632 , https://github.com/fossar/selfoss/pull/605 ) but my emails written in a different format, and until just not not associated with my github account - that is why I didn't show up here.
This might be the case for others, too, so we might have to extend that list by checking whether there are unassociated email addresses?

Anyway, for the actual change, I'm +1 for GPL3+.

**Comment by jtojnar** [comment]
@dubbl Right, I noticed that too, and started to compile a list of e-mails but cannot really post it here and have it on the other computer. Anyway, I made few GraphQL requests using the following query on https://developer.github.com/v4/explorer/ and got seven JSON documents.

```graphql
{
  repository(owner: "fossar", name: "selfoss") {
    pullRequests(first: 100, after: "…",) {
      nodes {
        author {
          login
        }
        url
        id
      }
      pageInfo {
        endCursor
        startCursor
        hasNextPage
      }
    }
  }
}
```

Then I slurped them using `cat contributions.json | jq -s 'map(.data.repository.pullRequests.nodes) | [.[][]] | group_by(.author) | map([.[0].author.login, [.[].url]])'` and got the [new list](https://gist.github.com/jtojnar/4cc575a930cb4fd3df74dac0364dd4d6). Unfortunately, I cannot include it into the opening issue since that many links break GitHub.

Hopefully, no one contributed using different method than pull requests or Weblate, I have no idea how to cross-reference it easily.

**Comment by Weishaupt** [comment]
I agree with licensing my contributions to selfoss under GNU GPL 3 or any later version.

**Comment by PhrozenByte** [comment]
> In order to clear up this uncertainty, we will need the past contributors to confirm if they are fine with licensing their contribution under GPL 3 or later

Yes, go ahead.

**Comment by seanrand** [comment]
Seems like I somehow missed the GitHub notification...

I agree with licensing all my contributions to selfoss under GNU GPL 3 or any later version.

**Comment by JFreakDK** [comment]
I am ok with licensing my contributions to selfoss under GNU GPL 3 as well.

**Comment by vincebusam** [comment]
I agree with licensing all my contributions to selfoss under GNU GPL 3 or any later version.

**Comment by ochristi** [comment]
Hi, I am fine with GPL3+, is this the right place to confirm?

**Comment by jtojnar** [comment]
@ochristi Yes, here is fine. Thank you.

**Comment by kereslas** [comment]
I agree with licensing my contributions to selfoss under GNU GPL 3 or any later version.

**Comment by squatica** [comment]
You can count me in, I agree with GPL3+

**Comment by gnumoksha** [comment]
I agree with GPL v3 or later
